README.md

```diff
@@ -48,13 +48,20 @@ conditions $X'$ with respect to a pairwise distance metric applied to existing e
 <br>
 
 (3) **Experiment Runners**: A method that orchestrates the collecting of observations for a given set of 
-experiment conditions.<br>
+experiment conditions (this may include the recruitment of participants).<br>
```

Comment:
```suggestion
experiment conditions, which may include the recruitment of participants.<br>
```

Reply:
sorry for the naivete, but to contribute an experiment runner is specification of the recruitment manager actually optional in some cases? or does something need to be declared, even if it's the equivalent of None/empty?

Reply:
It's a good point. People may contribute experiment runners without specifying recruitment managers (they would mostly be required for behavioral experiments).